modules/component-store/src/component-store.ts

```diff
@@ -244,7 +244,7 @@ export class ComponentStore<T extends object> implements OnDestroy {
     if (observables.length === 0) {
       observable$ = this.stateSubject$.pipe(
         config.debounce ? debounceSync() : (source$) => source$,
-        map(projector)
+        map((arg) => projector(arg))
```

Comment:
Thanks for creating this PR.
Let's give `arg` a name.

```suggestion
        map((state) => projector(state))
```

Reply:
Done, I got a bit too influenced by `projectorArgs` below.